docs/docs/03-usage/10-watermarker/03-java.md

```diff
@@ -18,64 +18,56 @@ If you want to use watermarking inside your Java project, this page gives you th
 information.
 
 ## Compile the Library
-See [Installation](../installation).
+See [Installation](01-installation.md).
 
 ## Example: Watermarking Text with Text
 Below you can see an example project that inserts a text as watermark into a cover text and then
 extracts the watermark from the watermarked text.
 
 ### 1. Add Library as Dependency
-*Line 11 is the important line that adds our library as a dependency into the project. Currently, we
-are working with local deployment, so you will have to add `mavenLocal()` as repo (line 8) and
-publish the library to mavenLocal (see [Installation](../installation)).*
-```kt title="build.gradle.kts" showLineNumbers
-plugins {
-    application
-}
-
-repositories {
-    mavenCentral()
-    mavenLocal()
-}
+Currently, we are working with local deployment, so you will have to publish the library to 
+mavenLocal (see [Installation](01-installation.md)).  
 
+Then you can import it to your local Java project using Gradle ... 
+```
 dependencies {
     implementation("de.fraunhofer.isst.trend:watermarker:0.1.0-SNAPSHOT")
 }
+```
 
-java {
-    toolchain {
-        languageVersion.set(JavaLanguageVersion.of(17))
-    }
-}
-
-application {
-    mainClass.set("Main")
-}
+... or Maven: 
+```
+<dependency>
+    <groupId>de.fraunhofer.isst.trend</groupId>
+    <artifactId>watermarker-jvm</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+</dependency>
 ```
 
 ### 2. Use the Library
-*The static functions `handle(...)` and `unwrap(...)` are optional for easy error handling with our
-custom return types (see [Concepts](../../../development/watermarker/concepts/#error-handling-1)
-for more details).*
+The essential methods of the library for inserting and extracting watermarks will generally 
+return a Result typed object. Result objects come with a flag that indicates if the operation 
+went successfully. They also contain a message field, which may be useful for testing and 
+debugging. And of they have a value field, from which the desired resulting payload can be 
+retrieved if the operation went successfully (and sometime even, if unsuccessful). The sample 
+code below is showing some more details on that. 
 
 *Watermark extraction can be customized for different use cases using optional Boolean parameters
 (see [Watermarker](../#extraction-customization) for more details)*
 
 ```java title="src/main/java/Main.java" showLineNumbers
 import de.fraunhofer.isst.trend.watermarker.Watermarker;
 import de.fraunhofer.isst.trend.watermarker.returnTypes.Result;
-import de.fraunhofer.isst.trend.watermarker.returnTypes.Status;
 import de.fraunhofer.isst.trend.watermarker.watermarks.TextWatermark;
 
 import java.util.List;
 
-
 public class Main {
     public static void main(String[] args) {
         // *********************
         // ***** INSERTION *****
         // *********************
-        
+
```

Comment:
Currently, all java code examples are shown in just one code block. Please split them up in multiple code blocks, starting with the first code block for one single watermark, the second code block for multiple watermarks etc.